.gitignore

```
@@ -16,6 +19,14 @@ ipyvuetify/labextension/
 ipyvuetify/nbextension/
 js/lib
 
-# readthedocs
-_build/
+# Sphinx documentation
+docs/_build/
+
+# Jupyter Notebook
+.ipynb_checkpoints
+
+# Unit test / coverage reports
+.ruff_cache
 
+# node installed dependencies
+**/node_modules/
```

```suggestion
**/node_modules/

```
Missing newline at end of file